src/main/configurations/Translate/Configuration_GenerateAuthorizationHeaderForCatalogiApi.xml

```diff
@@ -0,0 +1,86 @@
+<Module xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../FrankConfig.xsd">
+    <Adapter name="GenerateAuthorizationHeaderForCatalogiApi"
+        active="${GenerateAuthorizationHeaderForCatalogiApi.Active}"
+        description="">
+
+        <Receiver name="GenerateAuthorizationHeaderForCatalogiApi">
+            <JavaListener name="GenerateAuthorizationHeaderForCatalogiApi"  returnedSessionKeys="Error,Authorization" />
+        </Receiver>
+
+        <Pipeline>
+            <Exits>
+                <Exit name="EXIT" state="SUCCESS" />
+                <Exit name="EXCEPTION" state="ERROR" />
+            </Exits>
+
+            <XmlSwitchPipe 
+                name="CheckForJwt"
+                xpathExpression="$valueAuthType = 'jwt'"
+                >
+                <Param name="valueAuthType" value="${zaakbrug.zgw.catalogi-api.auth-type}"/>
+                <Forward name="true" path="createJwt"/>
+                <Forward name="false" path="CheckForBasicAuth"/>
+            </XmlSwitchPipe>
+
```

Comment:
If you are using a XmlSwitchPipe it would be better to just have `$valueAuthType` as xpathExpression and have forwards for `jwt, basic, value` instead of true and false. How it is used now would be more clear to the developer to use the XmlIf pipe instead. I think XmlSwitchPipe is the better fit in this case, as you can remove the other condition pipes further down.